Sprint-1/implement/max.js

```diff
@@ -1,4 +1,15 @@
 function findMax(elements) {
+  let max = -Infinity;
+  for (i = 0; i < elements.length; i++) {
+    if (typeof elements[i] === "number") {
+      if (elements[i] > max) {
+        max = elements[i];
+      }
+    }
+  }
+  return max;
 }
 
+// console.log(findMax(["l", "j", "o"]));
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```

Reply:
I’ve added 
 `if (typeof elements[i] === "number" && !isNaN(elements[i]))` to properly handle NaN values and ensure the function works correctly in all cases.
 
 -The function now correctly ignores NaN values and handles edge cases like [NaN] by returning -Infinity.